build_secrets.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import shutil
+from argparse import ArgumentParser, SUPPRESS
+import inspect
+
+
+# Get the current working directory
+APP_PATH = os.path.abspath(os.path.dirname(__file__))
+PACKAGE_NAME = "badtracksecrets"
+
+
+def custom_argparser():
+    """
+    Returns a custom argument parser which allows for named arguments to be required
+    """
+    # Get username and password
+    parser = ArgumentParser(add_help=False) # Disable default help
```

Comment:
Not a fan of argparse, but just a personal preference thing. Prefer environment variables for simplicity. Will accept anyway.